### DISCUSSION TITLE
As an Old StartSharp User I Can't Reach The Source Code

### DISCUSSION THREAD

**Discussion by muhammedyaman** [serenity-is/Serenity#7227] [General]
Hi. I am an old StartSharp user. I have license until v8.5. I need to check some pro extension source code for 6.6.3 project but i can't. Can I have access to it?

**Comment by VictorTomaili** [comment]
> If you choose to not renew, you'll only lose access to StartSharp source code, updates and premium support.
> You'll be keeping your StartSharp licence and continue using versions of StartSharp released during your subscription period.
> Your existing projects will continue to work, and you can still create new projects using your downloaded version of StartSharp.

https://serenity.is/docs/startsharp/README#:~:text=usually%20around%2025%25).-,If%20you%20choose%20to%20not%20renew%2C%20you%27ll%20only%20lose%20access%20to%20StartSharp%20source%20code%2C%20updates%20and%20premium%20support.,-You%27ll%20be%20keeping

**Comment by muhammedyaman** [comment]
Ok thanks for reminding. We will renew our license later on. Thanks for your perfect work and answer.
